### PR TITLE
fix: update quality-description and collection-description

### PR DESCRIPTION
**Pull request by MDavidson17**

### CODE REVIEW

File: topo_processor/stac/collection.py [hunk 1]
```diff
@@ -49,6 +50,7 @@ def __init__(self, title: str):
         # FIXME: Do we want to generate this id like this?
         self.id = str(ulid.ULID())
         self.title = title
+        self.description = "default description"
```
Comment (paulfouquet) [inline topo_processor/stac/collection.py:53]: Why is this needed? (instead of initializing with an empty string)

File: topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py [hunk 2]
```diff
@@ -121,6 +107,22 @@ def populate_item(self, metadata_row: Dict[str, str], asset: Optional[Asset] = N
         item.add_extension(StacExtensions.linz.value)
         item.add_extension(StacExtensions.version.value)
 
+    def population_collection(self, collection: Collection) -> None:
+        collection.description = "This aerial photographic survey was digitised from {} {} negatives in the Crown collection of the Crown Aerial Film Archive."
```
Comment (paulfouquet) [inline topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py:111]: Could this be all done in `collection.update_description()`? I get that this will be updated with the `colour` and `size` later but for understanding I would suggest to do everything at the same place.

**Review by paulfouquet — COMMENTED** [review]
That looks good. Just have a couple of questions

**Review by amfage — APPROVED** [review]

File: topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py [hunk 3]
```diff
@@ -121,6 +107,21 @@ def populate_item(self, metadata_row: Dict[str, str], asset: Optional[Asset] = N
         item.add_extension(StacExtensions.linz.value)
         item.add_extension(StacExtensions.version.value)
 
+    def population_collection(self, collection: Collection) -> None:
```
Comment (paulfouquet) [inline topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py:110]: As your function change the state of the `collection`, it's more clear to use a verb (i.e. `populate` instead of `population`) for the function name.

**Review by paulfouquet — CHANGES_REQUESTED** [review]
As @Megan is away I'll commit this quick change on her behalf.

**Review by paulfouquet — APPROVED** [review]
